ios/RNShare.m

```diff
@@ -249,7 +249,12 @@ - (NSDictionary *)constantsToExport
         if (@available(iOS 11.0, *)) {
             resolveBlock = successCallback;
             rejectBlock = failureCallback;
-            UIDocumentPickerViewController *documentPicker = [[UIDocumentPickerViewController alloc] initWithURLs:urls inMode:UIDocumentPickerModeExportToService];
+            UIDocumentPickerViewController *documentPicker = nil;
+            if (@available(iOS 15.0, *)) {
```

Comment:
Just in case people are using this on catalyst too, this is a bit more complete, but it's really the same idea of course

```suggestion
            if (@available(iOS 15.0, macCatalyst 15.0, *)) {
```